string object_name
AnnotationPt2D[] control_points
float32 left
float32 top
float32 width
float32 height

